eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventHubClientBuilder.java

```diff
@@ -132,18 +217,22 @@ public EventHubClientBuilder configuration(Configuration configuration) {
      * Use the default not null values if the Connection parameters are not provided.
      *
      * @return A new {@link EventHubClient} instance.
-     * @throws IllegalArgumentException when 'connectionString' is {@code null} or empty.
-     * @throws AzureException If the token provider cannot be created for authorizing requests.
+     * @throws IllegalArgumentException if the credentials have not been set using either {@link #credential(String)}
+     *         or {@link #credential(String, String, TokenCredential)}.
      */
     public EventHubClient build() {
         configuration = configuration == null ? ConfigurationManager.getConfiguration().clone() : configuration;
 
         if (credentials == null) {
-            String connectionString = configuration.get(AZURE_EVENT_HUBS_CONNECTION_STRING);
+            final String connectionString = configuration.get(AZURE_EVENT_HUBS_CONNECTION_STRING);
```

Comment:
Does this get the value for an environment variable by the name `AZURE_EVENT_HUBS_CONNECTION_STRING`? I don't believe any of the other languages support reading from env variables at the moment. Was this feature ported over from Track1?

Reply:
no. It was something we added. is there no mention of reading from env vars? i can rmeove it.

Reply:
You don't **have** to remove it, just don't document it until we have a consensus around it :) We can discuss this on Monday's sync up